### PR TITLE
Add RWX mention to descriptions

### PR DESCRIPTION
**Pull request by avestuk**
If the other PR (https://github.com/storageos/cluster-operator/pull/287) is approved then we should make identical changes here.

### CODE REVIEW

**Review by croomes — APPROVED** [review]
Wording is fine but it will need a version bump to release.

File: stable/storageos-operator/app-readme.md [hunk 1]
```diff
@@ -27,6 +29,8 @@ additional capacity, features and support plans contact sales@storageos.com.
 * High Availability - synchronous replication insulates you from node failure.
 * Delta Sync - replicas out of sync due to transient failures only transfer
     changed blocks.
+* Multiple AccessModes - dynamically provision ReadWriteOnce or ReadWriteMany
+  volumes
```
Comment (darkowlzz) [inline stable/storageos-operator/app-readme.md:33]: Please add a period at the end of the line.

File: stable/storageos-operator/README.md [hunk 2]
```diff
@@ -31,6 +33,8 @@ additional capacity, features and support plans contact sales@storageos.com.
 * High Availability - synchronous replication insulates you from node failure.
 * Delta Sync - replicas out of sync due to transient failures only transfer
     changed blocks.
+* Multiple AccessModes - dynamically provision ReadWriteOnce or ReadWriteMany
+  volumes
```
Comment (darkowlzz) [inline stable/storageos-operator/README.md:37]: It doesn't matters but please add one more level of indentation to maintain consistency with other points.

File: stable/storageos-operator/app-readme.md (hunk 1 above)
Comment (darkowlzz) [reply stable/storageos-operator/app-readme.md:33]: Please add one more level of indentation to maintain consistency with other points.

**Pull request merged**